imageio/imageio-psd/src/test/java/com/twelvemonkeys/imageio/plugins/psd/PSDImageReaderTest.java

```diff
@@ -104,7 +104,9 @@ protected ImageReaderSpi createProvider() {
                 // CMYK, uncompressed + contains some uncommon MeSa (instead of 8BIM) resource blocks
                 new TestData(getClassLoaderResource("/psd/fruit-cmyk-MeSa-resource.psd"), new Dimension(400, 191)),
                 // 3 channel, RGB, 32 bit samples
-                new TestData(getClassLoaderResource("/psd/32bit5x5.psd"), new Dimension(5, 5))
+                new TestData(getClassLoaderResource("/psd/32bit5x5.psd"), new Dimension(5, 5)),
+                // group layer psd file
+                new TestData(getClassLoaderResource("/psd/group_layer.psd"), new Dimension(4000, 1600))
```

Comment:
Unless this PSD file contains special features that needs general tests, you don't need to add it here. The specific test below will do! 😀